src/pretix/control/views/event.py

```diff
@@ -1504,3 +1506,35 @@ def formset(self):
                 },
             ] if self.request.method != "POST" else []
         )
+
+
+class EventQRCode(EventPermissionRequiredMixin, View):
+    permission = 'can_change_event_settings'
+
+    def get(self, request, *args, filetype, **kwargs):
+        url = build_absolute_uri(request.event, 'presale:event.index')
+
+        qr = qrcode.QRCode(
+            version=1,
+            error_correction=qrcode.constants.ERROR_CORRECT_M,
+            box_size=10,
+            border=4,
+        )
+        qr.add_data(url)
+        qr.make(fit=True)
+
+        if filetype == 'svg':
+            factory = qrcode.image.svg.SvgPathImage
```

Comment:
I do not know why, but importing `qrcode.image.svg` fails for me locally with error 

AttributeError: module 'qrcode.image' has no attribute 'svg'

If I `import qrcode.image.svg` at the top, it works. :/